src/makeRobotAccountant.js

```diff
@@ -6,7 +6,30 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (param) => {
```

Comment:
```suggestion
  return (param) => {
```
also you can directly return function to avoid naming

Reply:
good idea 🔥